### ISSUE TITLE
Call log modal focus order differs from visual order

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/call.zetk.in#270]
## Steps to reproduce

1. Open an assignment.
2. Click the log button.
3. Press `⌘ + F5` to start VoiceOver.
4. Focus the close button.

## Expected outcome

The close button is visually first so it should be first in the focus order. See [C27: Making the DOM order match the visual order](https://www.w3.org/TR/WCAG20-TECHS/C27.html) for details of the success criteria this addresses and the assistive technology use cases it supports.

## Actual outcome

The close button is at the end of the focus order despite being at the top left of the screen.

![VoiceOver focusing the close button at the end of the focus order](https://user-images.githubusercontent.com/566159/191597491-40c0faf1-2036-44ba-934b-31a9c71df5a8.gif)

## Severity

The success criterion covering this issue is WCAG level A – the minimum level. Most organizations aim for a conformance level of AA, which includes all level A criteria. In this particular case this issue seems unlikely to seriously hamper anybody's work but as it's level A it's still moderately severe.

## Recommendation

Rearrange the DOM order so that `.OverlayStack-closeButton` is before `.OverlayStack-overlay`. In React terms: just render the close button first.
